x/supplier/types/message_stake_supplier_test.go

```diff
@@ -76,14 +76,35 @@ func TestMsgStakeSupplier_ValidateBasic(t *testing.T) {
 			},
 		},
 		{
-			desc: "valid signer is neither the operator nor the owner",
+			desc: "valid signer is neither the operator nor the owner - empty service configs",
```

Comment:
Whos is the valid signer if it's neither the operator nor the owner?

I don't fully understand why they'd be able to sign the message without the error.